server/server.js

```diff
@@ -8,19 +8,15 @@ const MongoStore = require("connect-mongo")(session);
 const methodOverride = require("method-override");
 const flash = require("express-flash");
 const logger = require("morgan");
+//Use .env file in config folder
+require("dotenv").config({ path: "./config/.env" });
 const connectDB = require("./config/database");
 const mainRoutes = require("./routes/main");
 const eventsRoutes = require("./routes/events");
 
-//Use .env file in config folder
-require("dotenv").config({ path: "./config/.env" });
-
```

Comment:
Can you explain why this was move upward? Were issue with files not being able to access env variables?